### PR TITLE
Mention both spritezero and spreet

### PR DESCRIPTION
**Pull request by pka**
The referenced `spritezero` application is archived. `spreet` is actively maintained.

## Launch Checklist

 - [x] Confirm **your changes do not include backports from Mapbox projects** (unless with compliant license) - if you are not sure about this, please ask!
 - [x] Briefly describe the changes in this PR.
 - [ ] Link to related issues.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
 - [ ] Write tests for all new functionality.
 - [ ] Document any changes to public APIs.
 - [ ] Post benchmark scores.
 - [ ] Add an entry to `CHANGELOG.md` under the `## main` section.

### CODE REVIEW

**Comment by HarelM** [comment]
Is there something more popular? 
I'll see how they respond to the following request.
https://github.com/flother/spreet/issues/83

**Comment by pka** [comment]
I guess, spritezero ist still far more popular and I don't know of any other. We could also point to both tools?

**Comment by pka** [comment]
OSM Americana seems to use [@basemaps/sprites](https://github.com/linz/basemaps/tree/master/packages/sprites), despite the workflow graphics which contains `spritezero`.

**Comment by HarelM** [comment]
I think pointing to both tools at the moment might be a safer approach.

**Review by louwers — APPROVED** [review]

File: docs/sprite.md [hunk 1]
```diff
@@ -130,4 +130,4 @@ On high-DPI devices, `@2x` is appended to  the URLs described above. For example
 
 ## Generating Sprites
 
-A tool that can generate sprites from svg files can be found in here: [@elastic/spritezero](https://www.npmjs.com/package/@elastic/spritezero)
+A tool that can generate sprites from svg files can be found in here: [spreet](https://github.com/flother/spreet)
```
Comment (louwers) [inline docs/sprite.md:133]:
```suggestion
There are tools that can generate sprites from SVG files, such as [spreet](https://github.com/flother/spreet) and [spritezero](https://www.npmjs.com/package/@elastic/spritezero).
```

**Pull request merged**